supervisor/erratum_handler.py

```diff
@@ -131,6 +136,29 @@ def try_to_advance_erratum(self, new_status: ErrataStatus) -> WorkflowResult:
             )
 
         if rule_set.all_ok:
+            if new_status == ErrataStatus.REL_PREP:
+                # TODO: What is the best practice here to get package name?
+                build_nvr = get_erratum_build_nvr(self.erratum.id, "mypackage")
+                if not erratum_have_magic_string_in_comments(
+                    self.erratum.id, f"jotnar-product-listings-checked({build_nvr})"
+                ):
+                    prev_erratum = get_previous_erratum(self.erratum.id, "mypackage")
+                    if prev_erratum:
+                        if errata_have_same_file_lists(
+                            prev_erratum.id, self.erratum.id
+                        ):
+                            erratum_add_comment(
+                                self.erratum.id,
+                                f"jotnar-product-listings-checked({build_nvr})\n"
+                                "The product listings for this build match the previous build",
+                            )
+                        else:
+                            # TODO: What should we do if the file lists don't match?
+                            pass
+                    else:
+                        # TODO: What should we do if can't find the previous erratum?
```

Comment:
If there's no previous errata, then it's a new package, and we don't have to worry about the file lists changing, so just leave a comment with the magic string, and reschedule the task to move on to the next thing.